src/main.cpp

```diff
@@ -290,6 +292,19 @@ void process_file(padded_string_view input, error_reporter *error_reporter,
 }
 
 void print_help_message() {
+  const int max_width = 36;
+
+  auto print_helper = [&](const char *abbr, const char *message) {
+    std::cout << std::setw(max_width) << std::left << abbr << message << std::endl;
+  };
+
+  std::cout << "Usage: quick-lint-js [OPTIONS]... [FILE]..."
+            << std::endl << std::endl
+            << "OPTIONS" << std::endl;
+  print_helper("--debug-p, --debug-parser-visits", "Output travel when parsing file");
```

Comment:
Maybe don't bother documenting --debug options at all? They're for contributors, not users.